src/js/components/Video/StyledVideo.js

```diff
@@ -60,7 +60,7 @@ const headStyle = css`
     content: '';
     height: 100%;
     width: ${props => props.theme.global.edgeSize.xsmall};
-    background: ${props => normalizeColor('light-4', props.theme)};
+    background: ${props => normalizeColor(props.theme.video.scrubber.background, props.theme)};
```

Comment:
I think this property should be called `video.scrubber.color`. While we are using the `background` CSS property to show it, this is on the "head" of the scrubber, which should be the "color" of the scrubber.